src/build/__main__.py

```diff
@@ -42,6 +42,19 @@ def _error(msg, code=1):  # type: (str, int) -> None  # pragma: no cover
     exit(code)
 
 
+def _print_info(msg):  # type: (str) -> None
```

Comment:
Any reason this is private, and we could not allow the caller of the API (when called programmatically) to choose stdout/logging)?

Reply:
This is the private main implementation, the default one uses the logging module.